### PR TITLE
Implements call to WLAPI.FetchJWTBundle()

### PR DESCRIPTION
**Pull request by Andres-GC**

### CODE REVIEW

File: src/pyspiffe/workloadapi/default_workload_api_client.py [hunk 1]
```diff
@@ -165,14 +167,34 @@ def fetch_jwt_svid(
...
+            )
+
+            jwt_bundles = {
+                TrustDomain(td): JwtBundle.parse(TrustDomain(td), jwk_set)
+                for td, jwk_set in response.bundles.items()
+            }
+        except ParseJWTBundleError as pe:
+            raise FetchJwtBundleError(str(pe))
+        except Exception:
+            raise FetchJwtBundleError('JWT Bundles response is invalid')
```
Comment (maxlambrecht) [inline src/pyspiffe/workloadapi/default_workload_api_client.py:192]: Same comment about keeping the caught exception in the raised exception.

File: test/bundle/jwt_bundle/jwks/jwks_1_ec_key.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "keys": [
+        {
+            "kty": "EC",
+            "kid": "C6vs25welZOx6WksNYfbMfiw9l96pMnD",
+            "crv": "P-256",
+            "x": "ngLYQnlfF6GsojUwqtcEE3WgTNG2RUlsGhK73RNEl5k",
+            "y": "tKbiDSUSsQ3F1P7wteeHNXIcU-cx6CgSbroeQrQHTLM"
+        }
+    ]
+}
```
Comment (maxlambrecht) [inline test/bundle/jwt_bundle/jwks/jwks_1_ec_key.json:11]: nit: add new lines

**Review by maxlambrecht — COMMENTED** [review]
Nice work @Andres-GC! Left a couple of minor comments.

**Review by maxlambrecht — DISMISSED** [review]
💯

**Review by glaucimar — APPROVED** [review]
Thanks, Andres!

**Pull request merged**